src/hydrusvideodeduplicator/dedup.py

```diff
@@ -32,7 +32,7 @@ def __init__(self, client: hydrus_api.Client, verify_connection: bool = True):
         self.client = client
         if verify_connection:
             self.verify_api_connection()
-        self.hydlog.setLevel(logging.WARNING)
+        self.hydlog.setLevel(logging.INFO)
```

Comment:
?

Reply:
@ianwal --debug gives too much information (especially when it logs the massive GET requests for clearing out the database at the beginning), and--verbose gives nothing (cause we don't log shit to WARNING anyways). This was --verbose actually does something.

Reply:
Ah ok